product_variant_sale_price/README.rst

```diff
@@ -61,6 +61,7 @@ Contributors
 * Carlos Dauden <carlos.dauden@tecnativa.com>
 * Sergio Teruel <sergio.teruel@tecnativa.com>
 * Alex Comba <alex.comba@agilebg.com>
+* Fabien Bourgeois <fabien@yaltik.com>
```

Comment:
You should add this to readme/CONTRIBUTORS.rst instead, of this will be auto-removed. See https://github.com/OCA/maintainer-tools/#readme-generator

Reply:
Thanks. Done with 65ee7e5